src/components/Composer/index.native.js

```diff
@@ -1,12 +1,36 @@
 import React from 'react';
-import {StyleSheet} from 'react-native';
+import {Platform, StyleSheet} from 'react-native';
 import PropTypes from 'prop-types';
 import _ from 'underscore';
 import RNTextInput from '../RNTextInput';
 import themeColors from '../../styles/themes/default';
 import CONST from '../../CONST';
 import * as ComposerUtils from '../../libs/ComposerUtils';
 
+// Getting the commands module of the TextInput native component
+// See: https://github.com/facebook/react-native/blob/4a786d6b0d7a3420afdfb6b136d2ee3fa3b53145/Libraries/Components/TextInput/TextInput.js#L40
+let AndroidTextInputCommands;
+let RCTSinglelineTextInputNativeCommands;
+let RCTMultilineTextInputNativeCommands;
+if (Platform.OS === 'android') {
+    AndroidTextInputCommands = require('react-native/Libraries/Components/TextInput/AndroidTextInputNativeComponent').Commands;
+} else if (Platform.OS === 'ios') {
+    RCTSinglelineTextInputNativeCommands = require('react-native/Libraries/Components/TextInput/RCTSingelineTextInputNativeComponent').Commands;
+    RCTMultilineTextInputNativeCommands = require('react-native/Libraries/Components/TextInput/RCTMultilineTextInputNativeComponent').Commands;
+}
```

Comment:
These aren't documented on https://reactnative.dev/ are we safe to use those? They are used privately in the native TextInput component, so perhaps we aren't allowed to rely on such internal details 

If we are to use such, it might be best to submit a PR to react-native and add a `TextInputNativeCommands` API, similar to how there is https://reactnative.dev/docs/keyboard and other modules that deal with native handling

Reply:
> These aren't documented on [reactnative.dev](https://reactnative.dev/) are we safe to use those? They are used privately in the native TextInput component, so perhaps we aren't allowed to rely on such internal details

+1

Reply:
yeah, nw, i got rid of that